content/release-notes/1.52.1.md

```diff
@@ -0,0 +1,13 @@
+---
+date: 2021-10-04T08:00:00-07:00
+linktitle: "1.52.1"
+title: "1.52.1"
+kubernetes: "1.19, 1.20, and 1.21"
+---
+
+{{<fixes>}}
+ * Fixed which private images Kustomize targets inside of Helm charts when sub-charts are present.
+ * Changed preflight check logs to `info` level for progress messages and to `error` level for error messages.
+ * Fixed a bug that caused Admin Console memory usage to increase significantly while collecting support bundle.
```

Comment:
Fixed a bug that caused the Admin Console memory usage to increase significantly while collecting the support bundle.